runtime/doc/lsp.txt

```diff
@@ -71,12 +71,14 @@ options are not restored when the LSP client is stopped or detached.
 - |K| is mapped to |vim.lsp.buf.hover()| unless |'keywordprg'| is customized or
   a custom keymap for `K` exists.
 
-                                          *grr* *gra* *grn* *gri* *i_CTRL-S*
+                                 *grr* *gra* *grn* *gri* *gO* *i_CTRL-S*
 Some keymaps are created unconditionally when Nvim starts:
 - "grn" is mapped in Normal mode to |vim.lsp.buf.rename()|
 - "gra" is mapped in Normal and Visual mode to |vim.lsp.buf.code_action()|
 - "grr" is mapped in Normal mode to |vim.lsp.buf.references()|
 - "gri" is mapped in Normal mode to |vim.lsp.buf.implementation()|
+- "grs" is mapped in Normal mode to |vim.lsp.buf.workspace_symbol()|
```

Comment:
This line needs to be removed. Please check your PRs so that reviewers don't have to spend time on this.

Reply:
Sorry, I'll make sure to up my communication for the next time. 

I only have access to a phone for the next few days, but it's easier for me to make sure the PR is ready for review when I have access to my computer 

I will let you know when I do (I know its just 1 line change, but there could be some other problem I might not notice, so I don't want to waste your time)